pom.xml

```diff
@@ -300,7 +300,7 @@
             </eclipse>
 
             <importOrder>
-              <order>\#,\#org.alfresco,java,javax,jakarta,com,lombok,org,org.alfresco</order>
+              <order>\#java|\#javax,\#,\#org.alfresco,java|javax,,org.alfresco</order>
```

Comment:
`jakarta` is not a modern way to say `javax`? Maybe it should be located right after `javax`?

Reply:
Great suggestion - I'll update.